src/components/Typography.tsx

```diff
@@ -39,24 +40,46 @@ export const SectionHeading: FunctionComponent<SectionHeadingProps> = ({
 
 export const AnchorLink: FunctionComponent<AnchorLinkProps> = ({
   href,
+  description,
   children,
   classes = "",
 }: AnchorLinkProps): JSX.Element => (
   <a href={href} className={"govuk-link " + classes}>
-    {children}
+    <>
+      {children}
+      {!!description && (
+        <span
```

Comment:
Covers off: `A: Non-Descriptive Links in Context (2): On page /register-a-beacon/additional-beacon-use: Add visually hidden spans to make purpose of "change" and "delete" links clearer`